src/components/Data/Collections/Create.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div>
+    <create-or-update
+      headline="Create collection"
+      @collection-create::create="create"
+      :index="index">
+    </create-or-update>
+  </div>
+</template>
+
+<script>
+  import CreateOrUpdate from './CreateOrUpdate'
+  import { fetchCollectionDetail, resetCollectionDetail, createCollection } from '../../../vuex/modules/collection/actions'
+  import { collectionName } from '../../../vuex/modules/collection/getters'
+  import { indexesAndCollections } from '../../../vuex/modules/data/getters'
+
+  export default {
+    name: 'CollectionCreate',
+    props: {
+      index: String
+    },
+    components: {
+      CreateOrUpdate
+    },
+    vuex: {
+      actions: {
+        fetchCollectionDetail,
+        resetCollectionDetail,
+        createCollection
+      },
+      getters: {
+        collectionName,
+        indexesAndCollections
+      }
+    },
+    methods: {
+      create (name, mapping, isRealtime) {
+        this.createCollection(this.indexesAndCollections[this.index], this.index, name, mapping, isRealtime)
+          .then(() => {
+            this.$router.go({name: 'DataIndexSummary', params: {index: this.index}})
+          })
+          .catch((e) => {
+            this.$dispatch('toast', e.message, 'error')
```

Comment:
I'm not sure about error handling here: 
If the mapping is wrong it will be difficult to read the error in a toast.